pages/button/button.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title> button </title>
+    <link rel="stylesheet" href="button.css">
+</head>
+<body>
+   
+    <div class="join">
+        <h6>Join group</h6>
+    </div>
+    <div class="sign">
+        <h6>Sign up</h6>
+
+    </div>
+    <div class="learn">
+        <h6>Learn more</h6>
+    </div>
+    <div class="newchat">
+        <h6>   <img class="image" src="plus.png" alt=""> Suggest  New Chat</h6>
+       
+    </div>
+    <div class="joined">
+        <h6> <img class="image" src="joined.jpg" alt=""> Joined</h6>
+    </div>
+    <div class="addfreind">
+        <h6> <img class="image" src="addfr.png" alt=""> Add Freind</h6>
+    </div>
+    <div class="drop">
+        <div class="drp"><img src="drop.png" alt=""></div>
```

Comment:
What is `drp`?

Please rename it to a meaningful name that most of us can understand.
Simple is the better.

Reply:
Now i change the class name and make the meaning name